### PR TITLE
Nk solo start

### PR DESCRIPTION
**Pull request by nabilk11**

### CODE REVIEW

File: src/app/login/page.tsx [hunk 1]
```diff
@@ -1,3 +1,75 @@
...
+    e.preventDefault();
+    if (!email || !password) {
+      setError("Email and password are required!");
+      return;
+    }
+  };
+
+  return (
+    <div className="flex min-h-screen flex-col relative bg-slate-900 text-white">
+      <Header />
```
Comment (mmalamm) [inline src/app/login/page.tsx]: perhaps put this `<Header />` into a nested layout component? ie `src/app/login/layout.tsx`

File: src/app/login/page.tsx [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+        </button>
+        <h2>
+          Don&#39;t have an account?
+          <Link href={"/register"} className="text-orange-400">
+            {" "}
+            Register Here.
+          </Link>
+        </h2>
+      </main>
+      <Footer />
```
Comment (mmalamm) [inline src/app/login/page.tsx]: perhaps put this `<Footer />` into the layout.tsx also

File: src/app/page.tsx [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
+import Header from "@/components/Header";
 import Image from "next/image";
 
 export default function Home() {
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
-      <h1>Hello Run Club</h1>
-    </main>
+    <div className="flex min-h-screen flex-col relative bg-slate-900 text-white">
+      <Header />
```
Comment (mmalamm) [inline src/app/page.tsx]: maybe part of layout?

File: src/app/register/page.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        </button>
+        <h2>
+          Already have an account?
+          <Link href={"/login"} className="text-orange-400">
+            {" "}
+            Login Here.
+          </Link>
+        </h2>
+      </main>
+      <Footer />
```
Comment (mmalamm) [inline src/app/register/page.tsx]: layout.tsx

File: src/app/login/page.tsx (hunk 2 above)
Comment (nabilk11) [reply src/app/login/page.tsx]: fixed - moved header & footer to rootlayout for now and will render them conditionally

File: src/app/login/page.tsx (hunk 1 above)
Comment (nabilk11) [reply src/app/login/page.tsx]: fixed - moved header & footer to rootlayout for now and will render them conditionally

File: src/app/page.tsx (hunk 3 above)
Comment (nabilk11) [reply src/app/page.tsx]: fixed - moved header & footer to rootlayout for now and will render them conditionally

File: src/app/register/page.tsx (hunk 4 above)
Comment (nabilk11) [reply src/app/register/page.tsx]: fixed - moved header & footer to rootlayout for now and will render them conditionally

**Pull request merged**